app/services/index.js

```diff
@@ -1,2 +1,3 @@
-export { default as voteService } from './topics';
+import voteService from './topics';
```

Comment:
I actually like having an explicit `export { default as x } from a` statement, and I've seen it in a couple of places. 

If this is part of the eslint config, perhaps we can remove that rule